### PR TITLE
Improved goalkeeper

### PR DESCRIPTION
**Pull request by ultrainstinct30**
run_goalie.py works now and should be able to block some shots now.
Problems : the goalie bot stops responding sometimes randomly while in execute_. It again starts to respond if it is relocated.

### CODE REVIEW

**Comment by thebhatman** [comment]
This PR his pulled in unwanted commits from the previous PR you sent as well. This is supposed to have only commits related to the Goalie. Not the bsserver

**Comment by ultrainstinct30** [comment]
The latest commit includes the bsserver changes only in the goalie files, and the other mentioned changes in goalie and MotionPlanner.h

File: tactics/Goalie.py [hunk 1]
```diff
@@ -121,28 +189,30 @@ def execute_peace(self):
...
-		# self.ktp.spin()
+		state = None
+		try:
+			state = getState(state)
+		except rospy.ServiceException, e:
+			print e
+		if state:
+			state = state.stateB
+		self.ktp = KickToPoint.KickToPoint(target)
+		self.ktp.add_kub(self.kub)
```
Comment (thebhatman) [inline tactics/Goalie.py]: Do you think you might need to update kub state before adding kub?

Comment (thebhatman) [reply tactics/Goalie.py]: If yes, then make sure you do it and update kub state everywhere, before adding it to kicktopoint.

**Comment by thebhatman** [comment]
Resolve the merge conflicts.

**Pull request merged**